src/settings/Settings.js

```diff
@@ -150,6 +150,11 @@ export const SETTINGS = {
         displayName: _td('Autoplay GIFs and videos'),
         default: false,
     },
+    "hideInlineEncryptionIconUntilHover": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td('Hide inline encryption icons except when hovering over a message'),
```

Comment:
For UX, settings should probably be positive-sounding to avoid a confusing mess of toggles. The current state is all over the place, but some kind of standard direction would help a lot.

Reply:
Would something like "Display inline encryption icons only when hovering over a message" get us closer to the desired tone?

Reply:
"Always show encryption icons" (defaulting to true) is more along the lines of what I was thinking.

Reply:
Gotcha. Done.